docs/rpc-spec.md

```diff
@@ -374,6 +374,7 @@ The 'source' column here corresponds to the data structure there.
 | `announceState`           | number     | tr_tracker_view
 | `announce`                | string     | tr_tracker_view
```

Comment:
hmm, unrelated but just noticed that this is not in alphabetical order.  "announce" should come before "announceState"